blockonomics-woocommerce.php

```diff
@@ -60,6 +60,12 @@ function blockonomics_woocommerce_init()
     add_action('woocommerce_email_customer_details', 'nolo_bnomics_woocommerce_email_customer_details', 10, 1);
     add_filter('woocommerce_payment_gateways', 'woocommerce_add_blockonomics_gateway');
     add_filter('clean_url', 'bnomics_async_scripts', 11, 1 );
+    add_action('init', 'custom_rewrite_rules');
+
+    /* Custom Rewrite Rule */
+    function custom_rewrite_rules() {
```

Comment:
Let us  _show_order_ . Also don't hardcode `wc-api=WC_Gateway_Blockonomics` let us use `WC()->api_request_url` else it won't work for WC < 3.0

Reply:
We cannot use `WC()->api_request_url` because add_rewrite_rule requires the url to start with `index.php`. All wordpress urls can be accessed in this manner.
In later versions of woocommerce, they added their own rewrite rules to change the url, however, we still need to access the original page with the form `index.php?wc-api=WC_Gateway_Blockonomics`